cmd/argocd-server/commands/argocd_server.go

```diff
@@ -211,6 +211,11 @@ func NewCommand() *cobra.Command {
 				}
 			}
 		},
+		Example: `# Start the Argo CD API server with default settings
+		$ argocd-server
+		# Start the Argo CD API server on a custom port and enable tracing
+		$ argocd-server --port 8888 --otlp-address localhost:4317
```

Comment:
We should remove this extra space from the start of the line. It would useful to use `templates.Example`. Check https://github.com/argoproj/argo-cd/blob/ddce148f666cefdeab01fcffa43df71bd7a0f2a8/cmd/argocd/commands/account.go#L36-L48